### ISSUE TITLE
gitgraph-react fails to produce the image described in the readme.md file

### ISSUE THREAD

**Issue by erickhagstrom** [nicoespeon/gitgraph.js#351]
**Describe the bug**

Following the directions at https://github.com/nicoespeon/gitgraph.js/tree/master/packages/gitgraph-react, one expects to see on one's browser essentially the same image as displayed in the readme.md. Instead, one sees a similar image, but with commit messages, hashes and author aligned at the left margin and obscured by the graph itself.

The image displayed is below.

**To Reproduce**

The steps necessary to reproduce are already given in the gitgraph-react readme.md file.

**Expected behavior**

To see the image displayed in the readme.md file.

**Screenshots**

<img width="652" alt="Screen Shot 2020-04-01 at 12 23 09 PM" src="https://user-images.githubusercontent.com/12677760/78163356-4c835780-7416-11ea-8cb9-90f5343bd18b.png">

**Context**

- Which library is impacted? @gitgraph/react
- Which version of the library is it? 1.5.3
- What browser and version is impacted? Chrome Version 80.0.3987.132 (Official Build) (64-bit), running on MacOS Catalina v 10.15.3

**Additional information**

It appears that there was a commit about 13 months ago, changing the logic behind translating commit messages. https://github.com/nicoespeon/gitgraph.js/commit/ef0a6abbb36e391783e136bc153a0a117a5048d4

This seems to have caused gitgraph-react to behave differently than advertised on the readme.md.

I personally would prefer to see gitgraph-react to behave as advertised, i.e. please don't change the readme.md to match the code, but change the code to match the readme.md.

**Comment by nicoespeon** [comment]
Good catch, thanks @erickhagstrom 

If anyone is willing to deal with this, I'd advise to create a new story in the react Storybook: https://github.com/nicoespeon/gitgraph.js/blob/master/packages/stories/src/gitgraph-react/1-basic-usage.stories.tsx

Use the same code that the README is presenting, see the problem and fix it with the indication provided. That way, we'll be sure that the README example stays valid.

**Comment by Dachaz** [comment]
As I ran into the same thing today, I decided to investigate.

First, I tried using an older version of `@gitgraph/react`  from npm, but was surprised that the only one that rendered expected results was `1.0.0-1`. Finding it weird that nobody else spotted the react version being fundamentally broken this whole time, I went on to fork/clone and follow @nicoespeon 's advice to add the example to the storybook. When I did that: it worked:
<img width="1152" alt="Screenshot 2020-05-21 at 16 32 54" src="https://user-images.githubusercontent.com/124358/82569583-bd2a3300-9b80-11ea-88ae-d002fe6f70ae.png">

This brought up the question: why does it work there, and not on the dummy page I was testing with (and how come @erickhagstrom has the same issue? Enter `create-react-app`. If you remove `<React.StrictMode>`, the graph renders as expected.

So, this might hint that `@gitgraph/react` will have compatibility issues with the next major version of React (or just that it clashes with the strict mode analyser). For this, someone more experienced with React than me should take a deeper dive — but for others, just don't wrap the `<GitGraph>` component with `<React.StrictMode>`.
